frontend/src/components/common/EditStudent.vue

```diff
@@ -768,17 +767,17 @@ export default {
         return '';
       }
     },
-    getAssignedPen(assignedPen, studentPen){
-      if (!assignedPen) {
+    getAssignedPen(assignedPen, studentPen, penMatchResult){
+      if (!assignedPen && !penMatchResult) {
         return 'waiting on fixes';
```

Comment:
Capitalize please

Reply:
You want PenMatchResult? Most things in the file are camelCase